xcode/Subconscious/Shared/Config.swift

```diff
@@ -24,10 +24,6 @@ struct Config: Equatable, Codable {
 
     var untitled = "Untitled"
 
-    /// Toggle scratch note suggestion feature
-    var scratchSuggestionEnabled = true
```

Comment:
No longer need scratch feature, since you can create notes with no title/slug now.